test/libsolidity/syntaxTests/abiEncoder/passing_internal_type_to_abi_encodecall.sol

```diff
@@ -0,0 +1,14 @@
+struct S {
+    function () f;
+}
+
+contract C {
+    function f(uint) external {}
+
+    function main() external view {
+        S memory s;
+        abi.encodeCall(this.f, (s));
+    }
+}
```

Comment:
Could you add another test that goes over more types? We're clearly missing coverage here. It would be good to have one that tries as many different types as possible.